detox/src/logger/utils/customConsoleLogger.js

```diff
@@ -25,20 +25,20 @@ function getStackDump() {
 
 function proxyLog(bunyanLoggerFn) {
   return (...args) => {
-    bunyanLoggerFn({ event: 'USER_LOG' }, getOrigin(), '\n', util.format(...args));
+    bunyanLoggerFn({ origin: getOrigin() }, util.format(...args));
```

Comment:
I should verify again how exactly `console.*` overrides work compared to `master` branch, and if everyone in our team is happy.

Reply:
Overall, not much has changed. I added an extra script `npm run integration:console` to be able to manually see how the various console.* method calls look like when Detox overrides it.